sample/msleep_pages/rangeSelection.page.kts

```diff
@@ -1,6 +1,6 @@
-import krangl.DataFrame
+//@file:FromPage("sleepData")
 import krangl.*
 
-val sleepData: DataFrame = fromPage("sleepData")
+val sleepData: DataFrmae by inject()
```

Comment:
Mispelled `DataFrame` here

Reply:
that would explain it crashing on me.. oops